Assets/MixedRealityToolkit.Providers/WindowsMixedReality/BaseWindowsMixedRealitySource.cs

```diff
@@ -131,6 +131,11 @@ private void UpdateSourceData(InteractionSourceState interactionSourceState)
 
                 IsRotationAvailable = interactionSourceState.sourcePose.TryGetRotation(out currentSourceRotation);
 
+                // We want the source to follow the Playspace, so fold in the playspace transform here to 
+                // put the source pose into world space.
+                currentSourcePosition = MixedRealityPlayspace.TransformPoint(currentSourcePosition);
+                currentSourceRotation = MixedRealityPlayspace.Rotation * currentSourceRotation;
```

Comment:
Does this end up no-oping when hands are involved?

Reply:
It feels like it's fixing up a controller scenario but also is involved in a relatively fundamental hand scenario, so I'm curious if that stuff is being fixed up some other way right now

---
In reply to: [301810770](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/5250#discussion_r301810770) [](ancestors = 301810770)

Reply:
`MixedRealityPlayspace` will be at the origin, unless teleportation is used. So, yeah, in most cases on HoloLens this will no-op, unless a dev decides to use teleportation on a HoloLens.

Reply:
These lines are currently being used on our grip data as well:
https://github.com/microsoft/MixedRealityToolkit-Unity/blob/415582cd3256b633e862c8db8c829844bd13d603/Assets/MixedRealityToolkit.Providers/WindowsMixedReality/BaseWindowsMixedRealitySource.cs#L217
https://github.com/microsoft/MixedRealityToolkit-Unity/blob/415582cd3256b633e862c8db8c829844bd13d603/Assets/MixedRealityToolkit.Providers/WindowsMixedReality/BaseWindowsMixedRealitySource.cs#L218

Reply:
Yep, I saw the other lines, I was just more concerned about modifications to some other path.